README.md

```diff
@@ -317,6 +317,26 @@ import icon from '@assets/icon';
 import App from './App';
 ```
 
+##### 7. Group subpath import style local aliases
+
+If you define non-relative aliases to refer to local files without long chains of `"../../../"` via subpath imports, you can include those aliases in your `importOrder` to keep them grouped with your local code.
```

Comment:
It might be nice to use this as an opportunity to teach devs about subpath imports as well, maybe with a link to the node.js docs.  And maybe we can reword this to be a bit less copy-paste from the aliases above?   I can take a crack at a "suggestion" if you'd like.

Reply:
Certainly please provide a suggestion 👍